docs/package.json

```diff
@@ -33,42 +33,37 @@
     "gatsby-plugin-react-svg": "^3.0.1",
     "gatsby-plugin-remove-trailing-slashes": "^3.1.0",
     "gatsby-plugin-sass": "^4.1.0",
-    "gatsby-plugin-svg-sprite": "^2.0.1",
     "gatsby-source-filesystem": "^3.1.0",
     "gatsby-transformer-yaml": "^3.1.0",
     "html-react-parser": "^1.4.5",
     "postcss": "^8.2.8",
     "prism-react-renderer": "^1.2.0",
     "prop-types": "^15.7.2",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
     "react-element-to-jsx-string": "^14.3.4",
     "react-helmet": "^6.1.0",
-    "react-live": "^2.3.0",
+    "react-live": "^3.0.0",
     "sass": "^1.49.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.0.0",
+    "@nciocpl/eslint-config-react": "^0.8.0",
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.5",
     "@typescript-eslint/eslint-plugin": "^4.18.0",
     "autoprefixer": "^10.3.1",
     "babel-preset-gatsby": "^1.10.0",
+    "core-js": "^3.9.1",
     "dangerously-set-html-content": "^1.0.9",
-    "eslint-config-airbnb": "^18.2.1",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-config-react-app": "^6.0.0",
+    "eslint": "^7.32.0",
     "eslint-loader": "^4.0.2",
-    "eslint-plugin-flowtype": "^5.4.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-mdx": "^1.11.0",
-    "eslint-plugin-prettier": "^3.3.1",
-    "eslint-plugin-react": "^7.22.0",
-    "eslint-plugin-react-hooks": "^4.2.0",
+    "eslint-plugin-mdx": "1.16.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^26.6.3",
-    "prettier": "^2.2.1",
-    "uswds": "^2.12.0"
+    "typescript": "^4.5.4",
+    "uswds": "^2.12.0",
```

Comment:
I guess we need to pin uswds so we don't run into shifting sands.
A clean install yields us:
```
uswds@^2.12.0:
  version "2.13.1"
  ```

Reply:
Pinned it to 2.12.0 which seems to be what is in the lock file. This happens to be a 2 version off of the 2.12 series and a total of 4 versions behind the last.